action.yml

```diff
@@ -34,21 +34,7 @@ runs:
 
         # required for the below API call
         GH_TOKEN: ${{ github.token }}
-      run: |
```

Comment:
Is moving this out related to what we are trying to do here? Could we make this a separate PR?

Reply:
yes and yes